### PR TITLE
BA-433 line consistency

### PR DESCRIPTION
**Pull request by Braedencraig**
Create consistency between all line dividers on the site colour wise.

### CODE REVIEW

File: web/wp-content/themes/borealis-2021/page-search.php [hunk 1]
```diff
@@ -155,7 +155,7 @@
         }
         ?>
     </section>
-    <div class="border-b border-shade-grey-500"></div>
+    <div class="border-b bordercolor--shade-grey-500"></div>
```
Comment (lisafolkerson) [inline web/wp-content/themes/borealis-2021/page-search.php:158]: should this -- be  - ?

File: web/wp-content/themes/borealis-2021/page-search.php [hunk 2]
```diff
@@ -134,16 +134,16 @@
...
                             ?>
-                                <li class="last:border-b-0 border-b border-shade-grey-500">
+                                <li class="last:border-b-0 border-b bordercolor--shade-grey-500">
                                     <?php echo pg_generate_main_search_result($post, $research_areas, $id); ?>
                                 </li>
                             <?php endforeach;  // End of Query Loop 
                             ?>
                         </ul>
-                        <div class="border-solid border-t border-shade-grey-500 <?php echo intval($Query->max_num_pages) > 1 ? '' : 'hidden' ?> hover:bg-primary-navy-400 hover:text-shade-white-400 transition-all duration-300 load-more">
+                        <div class="border-solid border-t bordercolor--shade-grey-500 <?php echo intval($Query->max_num_pages) > 1 ? '' : 'hidden' ?> hover:bg-primary-navy-400 hover:text-shade-white-400 transition-all duration-300 load-more">
```
Comment (lisafolkerson) [inline web/wp-content/themes/borealis-2021/page-search.php:146]: should this -- be  - ?

File: web/wp-content/themes/borealis-2021/page-search.php [hunk 3]
```diff
@@ -134,16 +134,16 @@
                     if (!empty($Query->posts)) : // Empty Query check. 
                     ?>
                         <button class="refresh-results hidden"><?php echo esc_html('Refresh Results') ?></button>
-                        <ul class="posts-listing border-shade-grey-500 border-t" data-page="1" data-research-areas="<?php echo esc_attr(implode(',', $research_areas)) ?>" data-total="<?php echo esc_attr($Query->max_num_pages) ?>" data-query="<?php echo esc_attr($query) ?>" data-posttype="<?php echo esc_attr($id) ?>">
+                        <ul class="posts-listing bordercolor--shade-grey-500 border-t" data-page="1" data-research-areas="<?php echo esc_attr(implode(',', $research_areas)) ?>" data-total="<?php echo esc_attr($Query->max_num_pages) ?>" data-query="<?php echo esc_attr($query) ?>" data-posttype="<?php echo esc_attr($id) ?>">
                             <?php foreach ($Query->posts as $post) : // Start of Query loop 
                             ?>
-                                <li class="last:border-b-0 border-b border-shade-grey-500">
+                                <li class="last:border-b-0 border-b bordercolor--shade-grey-500">
```
Comment (lisafolkerson) [inline web/wp-content/themes/borealis-2021/page-search.php:140]: should this -- be  - ?

File: web/wp-content/themes/borealis-2021/page-search.php [hunk 4]
```diff
@@ -134,16 +134,16 @@
                     if (!empty($Query->posts)) : // Empty Query check. 
                     ?>
                         <button class="refresh-results hidden"><?php echo esc_html('Refresh Results') ?></button>
-                        <ul class="posts-listing border-shade-grey-500 border-t" data-page="1" data-research-areas="<?php echo esc_attr(implode(',', $research_areas)) ?>" data-total="<?php echo esc_attr($Query->max_num_pages) ?>" data-query="<?php echo esc_attr($query) ?>" data-posttype="<?php echo esc_attr($id) ?>">
+                        <ul class="posts-listing bordercolor--shade-grey-500 border-t" data-page="1" data-research-areas="<?php echo esc_attr(implode(',', $research_areas)) ?>" data-total="<?php echo esc_attr($Query->max_num_pages) ?>" data-query="<?php echo esc_attr($query) ?>" data-posttype="<?php echo esc_attr($id) ?>">
```
Comment (lisafolkerson) [inline web/wp-content/themes/borealis-2021/page-search.php:137]: should this -- be  - ?

File: web/wp-content/themes/borealis-2021/page-search.php [hunk 5]
```diff
@@ -113,16 +113,16 @@
...
                             <?php foreach ($Query->posts as $post) : // Start of Query loop 
                             ?>
-                                <li class="last:border-b-0 border-b border-shade-grey-500">
+                                <li class="last:border-b-0 border-b bordercolor--shade-grey-500">
```
Comment (lisafolkerson) [inline web/wp-content/themes/borealis-2021/page-search.php:119]: should this -- be  - ?

File: web/wp-content/themes/borealis-2021/page-search.php [hunk 6]
```diff
@@ -113,16 +113,16 @@
                     if (!empty($Query->posts)) : // Empty Query check. 
                     ?>
                         <button class="refresh-results hidden"><?php echo esc_html('Refresh Results') ?></button>
-                        <ul class="posts-listing border-shade-grey-500 border-t" data-page="1" data-research-areas="<?php echo esc_attr(implode(',', $research_areas)) ?>" data-total="<?php echo esc_attr($Query->max_num_pages) ?>" data-query="<?php echo esc_attr($query) ?>" data-posttype="<?php echo esc_attr($id) ?>">
+                        <ul class="posts-listing bordercolor--shade-grey-500 border-t" data-page="1" data-research-areas="<?php echo esc_attr(implode(',', $research_areas)) ?>" data-total="<?php echo esc_attr($Query->max_num_pages) ?>" data-query="<?php echo esc_attr($query) ?>" data-posttype="<?php echo esc_attr($id) ?>">
```
Comment (lisafolkerson) [inline web/wp-content/themes/borealis-2021/page-search.php:116]: should this -- be  - ?

File: web/wp-content/themes/borealis-2021/page-search.php (hunk 1 above)
Comment (lisafolkerson) [reply web/wp-content/themes/borealis-2021/page-search.php:158]: Seems like an issue this file only 👍🏻

**Review by lisafolkerson — CHANGES_REQUESTED** [review]
Can you please confirm changes on page-search.php ? I think the search and replace failed there.

Also I know there was a lighter grey and a darker grey on the site. I know this makes consistent but I thought the lighter grey (200 not 500) was what had been asked for but I'm not sure I'm remembering correctly.

**Comment by Braedencraig** [comment]
They requested all lines to match whats currently on the homepage live https://live-borealis-ai.pantheonsite.io/ so thats why I just went with 500! @lisafolkerson

**Review by lisafolkerson — APPROVED** [review]
Awesome thank you!

**Pull request merged**